kore/src/main/scala/org/kframework/definition/outer-ext.scala

```diff
@@ -64,6 +64,7 @@ object FlatModule {
         val newImports = m.imports.map(f)
         val newM = new Module(m.name, newImports, m.localSentences, m.att)
         newM.checkSorts()
+        newM.checkTokenSorts()
```

Comment:
There are two locations where we call `checkSorts()`. Why are we calling `checkTokenSorts()` only once?

Reply:
Oh right, thanks for the reminder. I'll extract these two lines in a new function and place that in both places that call checkSorts()